testkit/javadsl/src/main/scala/com/lightbend/lagom/javadsl/testkit/ServiceTest.scala

```diff
@@ -215,7 +248,13 @@ object ServiceTest {
    * When the server is started you can get the service client and other
    * Guice bindings here.
    */
-  class TestServer(val port: Int, val app: Application, server: Server) {
+  class TestServer(
+                    val port: Int,
+                    val app: Application,
+                    server: Server,
+                    @ApiMayChange val portSsl: Option[Int] = None,
+                    @ApiMayChange val sslContext: Option[SSLContext] = None,
+                  ) {
```

Comment:
This is slightly different from the `scaladsl`. In `scaladsl` the ports are obtained using `testServer.playServer.{port,httpsPort}`. 

Here, instead, `TestServer` already had a `port` field so users would read is using `testServer.port`. Also, `server: Server` was not visible. I chose to _add_ `portSsl` marked as `@ApiMayChange` in case we eventually deprecate `port` and `portSsl` and make `server: Server` visible like in the `scaladsl` implementation.

Reply:
Hmm, this is wrong. It's `javadsl` and exposes `Option[Int]` instead of `Optional[something]`.